assets/scripts/src/lib/utils.js

```diff
@@ -477,6 +477,19 @@ export const getWidthOfInput = (input) => {
     testEl.style.width = 'auto';
     testEl.style.whiteSpace = 'pre';
 
+    if (document.body.contains(input) && window.getComputedStyle) {
+      const inputStyle = window.getComputedStyle(input);
```

Comment:
This is working on your example page as intended, if you set `<style>.choices__input
{ font-size: 20px; }</style>`. 
But it's not working on my own codebase. `getComputedStyle` gets defaults, instead of actual values. I have to do it in async e.g. in `setTimeout`, but I don't know why.